packages/ember-glimmer/lib/components/curly-component.js

```diff
@@ -14,14 +15,31 @@ export class CurlyComponentSyntax extends StatementSyntax {
   }
 }
 
-import assign from 'ember-metal/assign';
+function argsToProps(args) {
+  let attrs = args.named.value();
+  let attrKeys = Object.keys(attrs);
+  let merged = { attrs: {} };
+
+  for (let i = 0, l = attrKeys.length; i < l; i++) {
+    let name = attrKeys[i];
+    let value = attrs[name];
+
+    // Do we have to support passing both class /and/ classNames...?
```

Comment:
We needed this in another branch, we will come back to this (hopefully one of the ported tests will catch it)